jormungandr-integration-tests/src/common/data/settings.rs

```diff
@@ -0,0 +1,27 @@
+#![allow(dead_code)]
+
+extern crate serde_derive;
+use self::serde_derive::{Deserialize, Serialize};
+use crate::common::configuration::genesis_model::LinearFees;
+use jormungandr_lib::time::SystemTime;
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct Settings {
+    pub block0_hash: String,
+    pub block0_time: SystemTime,
+    pub curr_slot_start_time: SystemTime,
+    pub consensus_version: String,
+    pub fees: LinearFees,
+    pub max_txs_per_block: u8,
+}
```

Comment:
this is to decode the settings returned by the JCLI command?

I think it would be much better to have something in `jormungandr-lib` that can be used in both library.

(ping @CodeSandwich )

Reply:
I've checked and format of this JSON is not engraved in any DTO, it's built with a `json!` macro ad-hoc inside REST handler. `jormungandr-ingegration-tests` depends on `jormungandr` anyway, so we probably could just create a public DTO in `jormungandr::rest::v0::???` for everyone to use.

Reply:
@dkijania You can rebase onto `rest_settings_dto` to get that public DTO :)

Reply:
I've made some modifications because it looks like jormungandr-integration-tests cannot use structs from application. So i moved SettingsDto to jormungandr-lib

Reply:
I think it's because `jormungandr` is defined as a `dev-dependency` of `integration-tests`, I'm not sure if that's what we want, could be. Or maybe we should remove this dependency completely, not sure really.

Reply:
i tried to move it as a `dependency` but with the same outcome